x-pack/filebeat/input/http_endpoint/handler.go

```diff
@@ -64,18 +69,40 @@ type handler struct {
 }
 
 func (h *handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	txID := h.nextTxID()
+	h.log.Debugw("request", "url", r.URL, "tx_id", txID)
```

Comment:
I'm worried about debug logs in hot paths like that. If there is a high throughput of incoming requests and a user turns on debug logging, this can flood the logs. 

It's not a blocker, I just wanted to raise my concern and let you decide what is best for this input.

Reply:
I'm OK with this.